src/pages/explore_grants/apply.tsx

```diff
@@ -204,7 +208,7 @@ function ApplyGrant() {
 					rewardCurrencyCoin={rewardCurrencyCoin}
 					rewardCurrencyAddress={rewardCurrencyAddress}
 					workspaceId={workspaceId}
-					safeNetwork={safeChainId || defaultChainId.toString()}
+					safeNetwork={safeObj.chainId}
```

Comment:
Use optional chaining. There might be an instance where the safeObj might be undefined.

Reply:
This is not only for this, but at all places. To account for cases where the safe has not been added.